### ISSUE TITLE
Behind the Scenes - Preparing for first meeting

### ISSUE THREAD

**Issue by mikepromazo** [promazo/Content-Team#12]

**Comment by Lupecasas** [comment]
![how to make the most of your meetings.png](https://images.zenhubusercontent.com/60255135f48be0068cc94f69/ce30fca7-7b4f-4d2f-ad3f-a60b9842af6a)

Hey Mike, not sure if you can see this from this link, otherwise I will also email it to you. Note: This post can either be like this or in 3 slides with this being more of a cover page design. @mikepromazo

**Comment by mikepromazo** [comment]
Thanks Lupe.  Notes:

- There's a lot of text for 1 page.  Let's break it up into 2-3.  The first one should be attention-grabbing and the other 2 can be more text heavy
- Use our standard fonts throughout.  SF Pro, avenir, or roboto
- Please share on Figma so I can update the text/tips
- What did you do before, during, and after your first meeting that was helpful?  What did your mentor do?

@Lupecasas

**Comment by Lupecasas** [comment]
@mikepromazo Yes, I'm really just basing it off of my and my friend's experience with the 100k. I had @emilmudroncik look over the copy and made these.  And I have put it in Figma to edit :)

[https://app.zenhub.com/files/317392277/a4ad07f2-1978-4adb-885c-1d45a4f5e7c2/download](https://app.zenhub.com/files/317392277/a4ad07f2-1978-4adb-885c-1d45a4f5e7c2/download)

**Comment by Lupecasas** [comment]
That last comment may not have uploaded right, here is a pdf try. If not they are in Figma under "Lupe's content page". @mikepromazo 

[https://app.zenhub.com/files/317392277/df252298-f44c-4380-ad4d-94a5a36f820f/download](https://app.zenhub.com/files/317392277/df252298-f44c-4380-ad4d-94a5a36f820f/download)

**Comment by mikepromazo** [comment]
Great work!  I made a few edits in a copied version below the original on Figma.  There is one comment I added that needs to be addressed then we can post.

@Lupecasas @emilmudroncik
